src/mscorlib/src/System/Runtime/InteropServices/NativeLibrary.Unix.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    // Contains UNIX-specific logic for NativeLibrary class.
+    public unsafe sealed partial class NativeLibrary
+    {
+        // [DllImport] (NDirectMethodDesc::FindEntryPoint) doesn't allow lookup by ordinal on non-Windows.
+        private const bool AllowLocatingFunctionsByOrdinal = false;
+
+        // Per PInvokeStaticSigInfo::InitCallConv and GetDefaultCallConv, the default calling convention
+        // on Unix is cdecl.
+        private const CallingConvention FallbackCallingConvention = CallingConvention.Cdecl;
```

Comment:
This may require msdn doc update, since in MSDN(https://msdn.microsoft.com/en-us/library/system.runtime.interopservices.callingconvention(v=vs.110).aspx),  

It states that "The callee cleans the stack. This is the default convention for calling unmanaged functions with platform invoke."
--